client_src/js/lib/modules/viewUpdater.js

```diff
@@ -1000,17 +1000,15 @@ async function updateState(printer, clientSettings, view) {
       const tool = document.getElementById(`${printer._id}-spool-${i}`);
       if (printer.selectedFilament[i] !== null) {
         const filamentManager = await checkFilamentManager();
-        if (filamentManager) {
```

Comment:
Removed that pointless if function. Surely that was stoner laziness I'd be embarrassed if not XD